### ISSUE TITLE
css icons not visible in zk1.4.4build182

### ISSUE THREAD

**Issue by MrMontesa** [zikula/core#3280]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.4build182
| PHP Version       | 5.6.26

### Expected behavior
Display CSS images

### Actual behavior
CSS images like `class="z-icon-es-view"` are not visible any longer. Saw this in News headline. It does look like this:
![news-no-css-icons](https://cloud.githubusercontent.com/assets/600125/21011982/2120c874-bd54-11e6-8cd7-977d8620bbcf.png)

But should looks like this:
![news-with-css-icons](https://cloud.githubusercontent.com/assets/600125/21011979/1e7d269e-bd54-11e6-913f-0aa11dd2a269.png)

### Steps to reproduce
- install zk1.4.4build182
- install/activate News module and/or Contnet
- access ?module=news&type=user&func=view and check news nav-bar
- tested with Bootstrap & Andreas08 theme

**Comment by Guite** [comment]
Reason is that it searches the images in `/theme/combined_asset/css/images/` instead of `/images/`. Workaround: disable asset merger in `/app/config/custom_parameters.yml`.

**Comment by MrMontesa** [comment]
I can confirm that the workaround is working fine.

**Comment by Guite** [comment]
Can't reproduce this problem. With activated asset merger the pathes are correctly rewritten:

```
.z-icon-es-view {
    background: transparent url("/src/images/icons/extrasmall/windowlist.png") no-repeat scroll 0 50%;
}
```

**Comment by Guite** [comment]
@MrMontesa Can you try the latest build of 1.4.5 to verify that it works now?

**Comment by MrMontesa** [comment]
As long as asset_mamager.compine is set to true, the behavior is still the same on my testbed!
http://zk14.amc-idstein.de/?module=news&type=user&func=view
This is build31 as now.

**Comment by Guite** [comment]
@MrMontesa seems like it works if Zikula is installed in a subdirectory only. Can you please test the fix proposed in #3294 ? Just change it in your system, clear the cache and check if it works then.

**Comment by MrMontesa** [comment]
Just gave it a try and the fix seems to solve the issue. The CSS icons are displayed as they should.
